src/RealtimeChannel.ts

```diff
@@ -76,6 +81,7 @@ export default class RealtimeChannel {
   pushBuffer: Push[] = []
   presence: RealtimePresence
 
+  /** Topic name can be any string. */
```

Comment:
Should this be one line down? Right above `public topic: string`?

Reply:
@alaister I think you're right. Will address.

Reply:
https://github.com/supabase/realtime-js/pull/202/commits/0c567f9a9de1a4cc257229705ebfb4a8c85b2efe